Cards/CardBase.gd

```diff
@@ -89,4 +93,9 @@ func _handle_effects_queue(caster: Entity, base_target: Entity) -> void:
 
 
 func animation_hit(hit_target: Entity, card_effect: EffectData, caster: Entity) -> void:
+	if _targets_triggered_hits.has(hit_target):
```

Comment:
doesn't that mean we can't play more than one effect on a target ? since the list is cleared in the `on_card_play` only

Reply:
Oh yeah forgot about that, i'll fix it